app/javascript/hyacinth_ui_v1/components/projects/enabled_dynamic_fields/EnabledDynamicFieldForm.js

```diff
@@ -20,7 +20,7 @@ class EnabledDynamicFieldEdit extends React.Component {
   }
 
   componentDidMount() {
-    axios.all([this.getDynamicFieldGraph(), this.getEnabledDynamicFields()])
+    axios.all([EnabledDynamicFieldEdit.getDynamicFieldGraph(), this.getEnabledDynamicFields()])
```

Comment:
Why not use `this`?

Reply:
Oh, is it because it's a static method?

Reply:
Yes - eslint rules pushing me in different directions; I picked static.